indico/htdocs/js/indico/modules/category_management.js

```diff
@@ -83,4 +83,8 @@
             });
         }
     };
+
+    global.setupCategoryEventsList = function setupCategoryEventsList() {
```

Comment:
`setupCategoryEventList` would seem like a better name from the grammatical standpoint...